app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -166,11 +169,29 @@
     }
   }
 
+  &--cerulean-bg{
+    padding: $base-padding;
+    background: $cerulean;
+    color: $white;
+    @include clearfix;
+
+    label{
+      color: $white;
+    }
+  }
+
   &--grey-border{
     border: 2px solid $stone;
   }
 }
 
+.section-content-half{
```

Comment:
instead of creating a new block for this, can we make a modifier on the section-half that's in this file?